LCM/scripts/python3/StartDscConfiguration.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+
+# Standard library imports
+from subprocess           import Popen, PIPE
+from sys                  import argv
+from imp                  import load_source
+from os.path              import dirname, isfile, join, realpath
+from fcntl                import flock, LOCK_EX, LOCK_UN, LOCK_NB
+from OmsConfigHostHelpers import write_omsconfig_host_telemetry, write_omsconfig_host_switch_event, write_omsconfig_host_log, stop_old_host_instances
+from time                 import sleep
+
+pathToCurrentScript = realpath(__file__)
+pathToCommonScriptsFolder = dirname(pathToCurrentScript)
+
+helperLibPath = join(pathToCommonScriptsFolder, 'helperlib.py')
+helperlib = load_source('helperlib', helperLibPath)
+
+
+def main(argv):
+    """StartDscConfiguration"""
+
+    # Define method arguments and description
+    description = 'Starts the specified DSC configuration.'
+
+    parameters = {
+        'configurationmof' : {
+            'shortForm' : 'c',
+            'helpText' : 'The path to the configuration mof to start.',
+            'required' : True,
+            'action' : 'store'
+        },
+        'force' : {
+            'shortForm' : 'f',
+            'helpText' : 'Specifies that any current pending configuration should be forcibly removed before starting the new configuration.',
+            'required' : False,
+            'action' : 'store_true'
+        }
+    }
+
+    # Parse -configurationmof on its own for backwards compatibility
+    configmofArgument = None
+    if '-configurationmof' in argv:
+        configmofIndex = argv.index('-configurationmof')
+
+        try:
+            configmofArgument = argv[configmofIndex + 1]
+        except:
+            print 'StartDscConfiguration.py: error: Please provide a valid path argument for -configurationmof'
+            exit(1)
+
+        # Set the configuration mof parameter to no longer be required so it doesn't error in the arugment parser
+        parameters['configurationmof']['required'] = False 
+
+        # Remove -configurationmof and its argument from the list so it doesn't error in the arugment parser
+        argv.pop(configmofIndex)
+        argv.pop(configmofIndex)
+
+    # Check that we don't have two configuration mofs defined
+    if configmofArgument and parsedArguments.configurationmof:
+        print 'StartDscConfiguration.py: error: Two configuration mof arguments were found. Please provide only one.'
+        exit(1)
+    
+    if configmofArgument:
+        parsedArguments.configurationmof = configmofArgument
+
+    # Read the configuration mof
+    try:
+        configurationFile = open(parsedArguments.configurationmof, 'r')
```

Comment:
I think you will probably have to use codecs.open in python3. I found this issue when testing changes in python3 environment. Please check if these are working.